src/NHibernate/Util/ReflectionCache.cs

```diff
@@ -222,5 +222,13 @@ internal static class TypeMethods
 			internal static readonly MethodInfo GetTypeFromHandle =
 				ReflectHelper.FastGetMethod(System.Type.GetTypeFromHandle, default(RuntimeTypeHandle));
 		}
+
+		internal static class StringMethods
+		{
+			public static readonly MethodInfo EndsWith = ReflectHelper.GetMethodDefinition<string>(x => x.EndsWith(null));
+			public static readonly MethodInfo StartsWith = ReflectHelper.GetMethodDefinition<string>(x => x.StartsWith(null));
+			public static readonly MethodInfo Contains = ReflectHelper.GetMethodDefinition<string>(x => x.Contains(null));
+			public static readonly MethodInfo Concat = ReflectHelper.FastGetMethod(string.Concat, default(string), default(string));
```

Comment:
This `Concat` addition is used nowhere, I do not think it should be added just in case it is used later.

Reply:
It was used. I refacrored it out. Will remove.